docs/book/src/cronjob-tutorial/testdata/project/config/manager/manager.yaml

```diff
@@ -93,7 +120,16 @@ spec:
           requests:
             cpu: 10m
             memory: 64Mi
+        # Extra volumeMounts for the manager container
+        # volumeMounts:
+        #   - name: extra-config
+        #     mountPath: /etc/extra
```

Comment:
I do not know if that will work with the patches but is a good call I think we need a PR for that and e2e tests to ensure that we do not break the behaviour